test/e2e/azdiskschedulerextender_test.go

```diff
@@ -40,9 +40,10 @@ var _ = ginkgo.Describe("AzDiskSchedulerExtender", func() {
 		schedulerExtenderTests(false)
 	})
 
-	ginkgo.Context("[multi-az]", func() {
-		schedulerExtenderTests(true)
-	})
+	// TODO add support for multi-az
```

Comment:
Be sure the create an issue for this.

Reply:
I just created it: https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/issues/51